apps/testingaccessibility/src/components/pricing.tsx

```diff
@@ -231,6 +232,12 @@ export const Pricing: React.FC<PricingProps> = ({
             </fieldset>
           </form>
         )}
+        {!merchantCoupon && (
+          <SaleCountdown
+            coupon={formattedPrice?.defaultCoupon}
+            data-pricing-product-sale-countdown={index}
+          />
+        )}
```

Comment:
logically speaking I'd consider checking for the presence of the default coupon and not the absence of the merchant coupon here. same basic effect, but "positive" logic and would be more accurate. the "data" attribute here might use an alias at the prop level